conf_files/pocs.yaml

```diff
@@ -66,6 +66,22 @@ messaging:
     cmd_port: 6500
     msg_port: 6510
 
+########################## Observations ########################################
+# Observations consist of a sequence of images and are grouped together
+# according # to the time the sequence started and finish when the mount
+# slews to new coordinates. There will be one folder per camera per target
+# per observation. An example folder structure would be:
+#
+#   $PANDIR/images/fields/PAN001/Hd189733/14d3bd/20180901T120001/
+#
+# In this folder will be stored JPG and FITS images. A timelapse of the
+# observation can be made (on per camera) and the JPGs optionally removed
+# afterward.
+################################################################################
+observations:
```

Comment:
TODO: Add some fields to control cleanup in the future (e.g. after 30 days, if the images have been uploaded and the disk has less than 30GB free, then remove the oldest images until we have at least 30GB free).